src/ui/DynamicHierarchicalFacet/DynamicHierarchicalFacetValues/DynamicHierarchicalFacetValueRenderer.ts

```diff
@@ -39,23 +38,32 @@ export class DynamicHierarchicalFacetValueRenderer {
     this.button.toggleClass('coveo-with-space', this.shouldHaveMargin);
     this.button.toggleClass('coveo-show-when-collapsed', this.shouldShowWhenCollapsed);
     this.facetValue.isSelected && this.button.setAttribute('disabled', 'true');
-    this.shouldHaveArrow && this.prependArrow();
+    this.shouldHaveBackArrow && this.prependBackArrow();
+    this.shouldHaveForwardArrow && this.appendForwardArrow();
   }
 
-  private prependArrow() {
-    const arrowIcon = $$('div', { className: 'coveo-dynamic-hierarchical-facet-value-arrow' }, SVGIcons.icons.arrowDown);
-    SVGDom.addClassToSVGInContainer(arrowIcon.el, 'coveo-dynamic-hierarchical-facet-value-arrow-svg');
+  private prependBackArrow() {
+    const arrowIcon = $$('div', { className: 'coveo-dynamic-hierarchical-facet-value-arrow-left' }, SVGIcons.icons.arrowDown);
     this.button.prepend(arrowIcon.el);
   }
 
+  private appendForwardArrow() {
+    const arrowIcon = $$('div', { className: 'coveo-dynamic-hierarchical-facet-value-arrow-right' }, SVGIcons.icons.arrowDown);
+    this.button.append(arrowIcon.el);
+  }
+
   private get shouldHaveMargin() {
     return !this.facetValue.isSelected && this.facetValue.path.length > 1 && !this.facetValue.children.length;
   }
 
-  private get shouldHaveArrow() {
+  private get shouldHaveBackArrow() {
     return this.facet.values.hasSelectedValue && !this.facetValue.isSelected && !!this.facetValue.children.length;
   }
 
+  private get shouldHaveForwardArrow() {
+    return !this.facetValue.isLeafValue && !this.facetValue.isSelected && !this.facetValue.children.length;
```

Comment:
Why do we check the `isLeafValue` property and the number of children `children.length`? Does the api not handle this?

Reply:
I'm checking that the value has no children **currently** because I only want to show the > arrow on the last level. If the value has children, it's on an upper level

Here's the mockup:
<img width="609" alt="Screen Shot 2020-09-15 at 5 06 42 PM" src="https://user-images.githubusercontent.com/4923043/93265156-2eb46f80-f776-11ea-820c-84443cfbe3fa.png">

**Laptops** is not a leafValue, but has children, we don't want an arrow.
**2-in-1 Laptops** is not a leafValue, but it selected, we don't want an arrow.
**Lorem Ipsum** is a leafValue, we don't want an arrow.
**Vostro Series** is not a leafValue, is not selected, and has no children (currently), we want the > arrow 😄